hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/blockmanagement/BlockManager.java

```diff
@@ -5360,4 +5360,14 @@ public void disableSPS() {
   public StoragePolicySatisfyManager getSPSManager() {
     return spsManager;
   }
+
+  public void setExculeSlowDataNodesForWriteEnabled(boolean enable) {
```

Comment:
Sorry for the late reply. Please fix typo of method name. The other changes look good to me.

Reply:
@tomscut Thank you for your reply.
Fixed and updated PR.